src/components/pages/dashboard/DashboardTopArea.tsx

```diff
@@ -24,7 +24,7 @@ export const DashboardTopArea = (props: OwnProps) => {
           <div className="py-2 pl-12 pr-6 whitespace-nowrap w-56">System ID/Name</div>
           <div className="py-2 px-6 whitespace-nowrap w-52">Last Update Time</div>
           <div className="py-2 px-6 whitespace-nowrap flex-grow">Logs</div>
-          <div className="py-2 pl-6 pr-12 whitespace-nowrap w-128">Energy</div>
+          <div className="py-2 pl-6 pr-12 whitespace-nowrap w-128">Energy (kWh)</div>
```

Comment:
[Proposal] Instead of adding the same exact y axes label to all the graphs, why not just adding that label to the table header instead? I'm afraid using the same label in every single chart would look redundant at the same time that I would still space to display the chart. 

![image](https://user-images.githubusercontent.com/42070238/112181502-d8058200-8c3f-11eb-9735-ddead93d6696.png)

Reply:
Fixed in 8a26f18
![image](https://user-images.githubusercontent.com/42070238/112705155-d8707800-8ee0-11eb-9248-484b3a4b4287.png)

I'm noticing that the label looks a little bit pixelated in my screen. Researched about this and I found multiple opinions about it but not a clear solution to it. If this bothers you, I will investigate the issue a little bit more or directly make the scale label using CSS.